apps/web/src/server/views/applications/case-documentation/documentation-folder.njk

```diff
@@ -12,17 +14,28 @@
 		items: breadcrumbItems
 	}) }}
 
-	<h2 class="govuk-heading-l">{{ currentFolder.displayNameEn }}</h2>
+{# Display sub-folders in this folder #}
+	<div class="pins-dashboard-list--hide-controls">
+		<h2 class="govuk-heading-l">{{ currentFolder.displayNameEn }} folder</h2>
 
-	<!-- loop to display sub-folders in this folder -->
-	{% for subFolder in subFolders %}
-		<div class="govuk-grid-row govuk-grid-column-full">
-			<a class="govuk-link govuk-body-s govuk-!-font-weight-bold"
-			   href="{{ 'document-category'|url({caseId: caseId, documentationCategory: subFolder}) }}">
-				{{ subFolder.displayNameEn }}
-			</a>
-		</div>
-	{% endfor %}
+		{% set folderRows = [] %}
+		{% for subFolder in subFolders %}
+			{% set folderRows = folderRows | concat ([[
+				{ html: '<a class="govuk-link govuk-body-m" href="' + 'document-category' |url({caseId: caseId, documentationCategory: subFolder}) +'">' + subFolder.displayNameEn + '</a>' }
+			]])
+			%}
+		{% endfor %}
+
+		{{ govukAccordion({
```

Comment:
do you need an Accordion here? I can see in the CSS that the controls are then hidden